sno/gpkg.py

```diff
@@ -238,31 +292,44 @@ def geom_to_ogr(gpkg_geom, parse_srs=False):
     return geom
 
 
+def wkb_to_gpkg_geom(wkb, **kwargs):
+    ogr_geom = ogr.CreateGeometryFromWkb(wkb)
+    return ogr_to_gpkg_geom(ogr_geom, **kwargs)
+
+
+def hex_wkb_to_gpkg_geom(hex_wkb, **kwargs):
+    if hex_wkb is None:
+        return None
+
+    wkb = binascii.unhexlify(hex_wkb)
+    return wkb_to_gpkg_geom(wkb, **kwargs)
+
+
 # can't represent 'POINT EMPTY' in WKB.
 # The GPKG spec says we should use POINT(NaN, NaN) instead.
 # Here's the WKB of that.
-GPKG_WKB_POINT_EMPTY = b'\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\xF8\x7F\x00\x00\x00\x00\x00\x00\xF8\x7F'
+# We can't use WKT here: https://github.com/OSGeo/gdal/issues/2472
+WKB_POINT_EMPTY_LE = b'\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\xF8\x7F\x00\x00\x00\x00\x00\x00\xF8\x7F'
 
 
-def ogr_to_geom(ogr_geom):
+def ogr_to_gpkg_geom(
+    ogr_geom, _little_endian=True, _little_endian_wkb=True, _add_envelope=False
+):
     """
     Given an OGR geometry object, construct a GPKG geometry value.
     http://www.geopackage.org/spec/#gpb_format
 
     Arbitrarily, this only produces little-endian geometries.
```

Comment:
Presume the `_*` parameters are for testing? Though the docstring is wrong now.

Reply:
Will fix. Yes, the underscored kwargs are only for testing. I'll make them keyword only at least I guess, and update the docstring